code/modules/antagonists/traitor/backstory/traitor_backstories.dm

```diff
@@ -0,0 +1,124 @@
+
+/datum/traitor_backstory/legally_enslaved
+	name = "The Legal Slave"
+	description = "Shit... I signed a contract I shouldn't have. <b>Now they own me.</b> I have to do their bidding, and if I don't... \
+	<b>They'll come for me.</b>"
+	motivations = list(TRAITOR_MOTIVATION_FORCED, TRAITOR_MOTIVATION_DEATH_THREAT)
+
+/datum/traitor_backstory/savior
+	name = "The Savior"
+	description = "Nanotrasen are corrupt, evil to the core. The crew here are sheep. Cogs in a machine. \
+	I must liberate them by showing the error in their ways, and expose Nanotrasen for what they truly are."
+	allowed_factions = list(
+		TRAITOR_FACTION_SYNDICATE,
+		TRAITOR_FACTION_INDEPENDENT
+	)
+	motivations = list(TRAITOR_MOTIVATION_NOT_FORCED, TRAITOR_MOTIVATION_POLITICAL, TRAITOR_MOTIVATION_AUTHORITY)
+
+/datum/traitor_backstory/hater
+	name = "The Hater"
+	description = "Nanotrasen ruined my life. They ruined everything. They took the things that I love away from me. <b>Now I'm going to make them pay.</b>"
+	allowed_factions = list(
+		TRAITOR_FACTION_BLACK_MARKET,
+		TRAITOR_FACTION_SYNDICATE,
+		TRAITOR_FACTION_INDEPENDENT
+	)
+	motivations = list(TRAITOR_MOTIVATION_NOT_FORCED, TRAITOR_MOTIVATION_POLITICAL, TRAITOR_MOTIVATION_LOVE)
+	murderbone = TRUE
+
+/datum/traitor_backstory/greedy
+	name = "The Greedy"
+	description = "If I do this, I'll be set for life. I'll have everything I ever wanted, and more. \
+	The payment is astronomical, and I'm fit for the job. Let's do this."
+	allowed_factions = list(
+		TRAITOR_FACTION_BLACK_MARKET,
+		TRAITOR_FACTION_SYNDICATE,
+		TRAITOR_FACTION_INDEPENDENT
+	)
+	motivations = list(TRAITOR_MOTIVATION_NOT_FORCED, TRAITOR_MOTIVATION_MONEY)
+
+/datum/traitor_backstory/climber
+	name = "The Climber"
+	description = "Life is a ladder, and there is only climbing it. I shall have no friends, I shall hold no loyalties, \
+	for the only end goal in life is for my ego to be supreme. \
+	In my many years of observing the dynamics in this universe, it is clear to me that this is the surest way to achieve the domination of myself. \
+	Today marks the beginning of my ascent, nothing matters but my rise. I am supreme."
+	allowed_factions = list(
+		TRAITOR_FACTION_BLACK_MARKET,
+		TRAITOR_FACTION_SYNDICATE,
+		TRAITOR_FACTION_INDEPENDENT
+	)
+	motivations = list(TRAITOR_MOTIVATION_NOT_FORCED, TRAITOR_MOTIVATION_MONEY, TRAITOR_MOTIVATION_REPUTATION, TRAITOR_MOTIVATION_FUN)
+
+/datum/traitor_backstory/machine
+	name = "The Machine"
+	description = "I was born in the Syndicate. I was made in the Syndicate. I <b>am</b> the Syndicate. \
+	I am nothing without the Syndicate, and I will do <b>everything</b> I am asked."
+	allowed_factions = list(TRAITOR_FACTION_SYNDICATE)
+	motivations = list(TRAITOR_MOTIVATION_NOT_FORCED, TRAITOR_MOTIVATION_AUTHORITY)
+	murderbone = TRUE
+
+/datum/traitor_backstory/sadist
+	name = "The Sadist"
+	description = "I want power, not over people, but over life, to inflict pain and suffering is my road to power. \
```

Comment:
I don't like the bit at the end that emphasises kill, kill, kill. if they are going to have a backstory that encouarges capturing and tortuting people, thats fine but it should be worded carefully to make it sound like all the person wants is mindless violence. If they get pleasure from the suffering of others then that's good, but it should emphasise that rather than just raw killing.

Reply:
It's intended for murderbone but I can remove it

Reply:
I think that it can be non-murderbone but it needs to be clear that they get enjoyment out of the suffering of others and not murder.
Something along the lines of a non-murderbone sadist creating a game-show where 2 non-targets compete to the death, somewhere where upon death they can be easily recovered.

Reply:
..  I've done that EXACT gimmick. Lmao. I had a trivia show where I sent them to space with a spacesuit if they lost